doc/nrf/ug_creating_matter_accessory.rst

```diff
@@ -0,0 +1,267 @@
+Creating Matter Sensor application from Matter Template sample
+##############################################################
+
+| In this tutorial you will learn how to create your own device application using Matter. 
+| We will show a way to modify Matter Template sample in order to obtain a sensor application.
+
+Our goal is to create a sensor, which, if active, periodically updates measured value.
+
+
+Matter Template sample
+**********************
+
+| We will base our sensor device on a Matter Template sample which provides a basic implementation of a Matter accessory.
+| Our first step in creating sensor device will be to copy content from :file:`samples/matter/template` to another place eg. :file:`samples/matter/sensor`.
+
+From now on, paths will be given with respect to :file:`samples/matter/sensor` directory.
+
+Application Clusters
+********************
+
+To add new functionalities to a Matter sample we need to add new application clusters.
+
+Cluster is a representation of a single functionality within a Matter accessory.
+Each cluster contains attributes which are stored in device's memory, and commands that can be used to modify or read the state of those attributes.
+
+Information about changes in cluster's state is passed using a ZCL callback. It can be used to alter device's behaviour when state of a cluster is changing
```

Comment:
In .rst files we have a one sentence - one line rule. So every sentence should be written in one line without newlines and another one started from a new line.